### ISSUE TITLE
Invert text/exposé positions in amendments sidebar ?

### ISSUE THREAD

**Issue by RouxRC** [regardscitoyens/the-law-factory#141]

**Comment by RouxRC** [comment]
@teymour was proposing it
I feel like both ways have advantages
Other views @fmassot  @njoyard ?

**Comment by njoyard** [comment]
I'm not sure, personally I prefer having exposé first to have a better clue what the amendment is about, but having text first makes comparing similar amendments easier...

**Comment by fmassot** [comment]
well, IMHO it depends on your law knowledge : if I know the law very well, then I would prefer to see the text first. But if I know almost nothing about the law, I would prefer to read the expose as I would not understand the text.

so finally it really depends on the user. Maybe we should keep the exposé first as we want to target a global audience.

**Comment by RouxRC** [comment]
i guess that was the original intention, then maybe we could have some kind of way of always showing half expose half amendment, with a click needed to unfold the full expose when necessary

**Comment by fmassot** [comment]
indeed, exposé can be quite long (and uninteresting :)) so maybe we can just show the first part (for example 250 characters) and let the user make a click to read the full exposé. As the text comes after, we can just show the full text

**Comment by njoyard** [comment]
This would be far easier to make after #152. Can we postpone it?

**Comment by fmassot** [comment]
+1 for leaving this feature after #152

**Comment by boogheta** [comment]
After discussion with @teymour, we both agreed hiding part of the exposé makes sense, although, one important thing will be to let the text of the amendment always start at the same position so that you can keep your eye on the same level when switching from one to another. This means the Signataires part will also need to be handled together with the exposé

**Comment by njoyard** [comment]
Re-fixed in fa8318c :)

**Comment by boogheta** [comment]
I think we're nearly there, although I find the space allocated for the signataires way too big, only the height of 2 lines should be given as it is not important, can always be unfolded, and mostly leaves a big weird white space before we can read the actually interesting info

**Comment by boogheta** [comment]
A few more things:
- unfolded text should not use the hand pointer but a regular one as it is not clickable but actually selectable
- when we open for the first time an amendment, the display is one that there is no way to get back to, which is a bit confusing. It makes sense as we want to keep the unfolded zone selectable for copy-paste for instance, but maybe the initial state should always be with one of the three (probably exposé unfolded by default). Another option would be to allow the top part of the bar to be clickable (with the hand then) to refold everything.
- not sure whether it is the case already, but if not I would keep in localstorage the display choice so it can apply to all texts for a same user would be nice

**Comment by njoyard** [comment]
All fixed :)

**Comment by RouxRC** [comment]
strangely I still have the hand cursor on the unfolded zone on your dev instance

**Comment by njoyard** [comment]
I do, too, maybe the travis job takes more time than usual to deploy.

**Comment by njoyard** [comment]
It's deployed correctly now !

**Comment by boogheta** [comment]
mmm the new css still won't load on my machine but yes when i modify it it works, maybe we could try and force the css to be hard refreshed in some way? although i don't know any proper way to do that :/

**Comment by njoyard** [comment]
NOW it's finally fixed

**Comment by boogheta** [comment]
yeah!!!
